packages/@webex/webex-core/src/lib/credentials/credentials.js

```diff
@@ -258,8 +258,28 @@ const Credentials = WebexPlugin.extend({
    * @returns {Promise<Token>}
    */
   downscope(scope) {
+    const superTokenScopeArr = this.supertoken.scope.split(' ');
```

Comment:
These token scope calculations look like they belong in '.scope' with the other scope utils